drivers/SmartThings/zigbee-button/src/zigbee-multi-button/heiman/init.lua

```diff
@@ -39,12 +40,14 @@ local is_heiman_button = function(opts, driver, device)
 end
 
 local do_configure = function(self, device)
-  device:send(device_management.build_bind_request(device, PowerConfiguration.ID, self.environment_info.hub_zigbee_eui))
-  device:send(PowerConfiguration.attributes.BatteryVoltage:configure_reporting(device, 30, 21600, 1))
-  for endpoint = 1,HEIMAN_NUM_ENDPOINT do
-    device:send(device_management.build_bind_request(device, OnOff.ID, self.environment_info.hub_zigbee_eui, endpoint))
+  if device:get_model() == "SceneSwitch-EM-3.0" then
+    device:send(device_management.build_bind_request(device, PowerConfiguration.ID, self.environment_info.hub_zigbee_eui))
+    device:send(PowerConfiguration.attributes.BatteryVoltage:configure_reporting(device, 30, 21600, 1))
+    for endpoint = 1,HEIMAN_NUM_ENDPOINT do
+      device:send(device_management.build_bind_request(device, OnOff.ID, self.environment_info.hub_zigbee_eui, endpoint))
+    end
+    device:send(OnOff.attributes.OnOff:configure_reporting(device, 0, 600, 1))
   end
```

Comment:
the DTH seems to send all of these commands to all of the Heiman devices.

Reply:
@greens in [DTH code](https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/zigbee-scene-keypad.src/zigbee-scene-keypad.groovy) for 'HS6SSB-W-EF-3.0', there is no battery capability. So @liuyd96 added a condition. 
FYI, the only device in this driver(SceneSwitch-EM-3.0) is in difference [DTH](https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy) which have battery capability.

Reply:
> @greens in [DTH code](https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/zigbee-scene-keypad.src/zigbee-scene-keypad.groovy) for 'HS6SSB-W-EF-3.0', there is no battery capability. So @liuyd96 added a condition. FYI, the only device in this driver(SceneSwitch-EM-3.0) is in different [DTH](https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy) which have battery capability.

So I added HS6SSB-W-EF-3.0 to wrong EdgeDriver ?